source/slang/slang-parameter-binding.cpp

```diff
@@ -1065,81 +1075,87 @@ static void addExplicitParameterBindings_GLSL(
     // the index/offset/etc.
     //
 
-    TypeLayout::ResourceInfo* resInfo = nullptr;
-    TypeLayout::ResourceInfo* foundResInfo = nullptr;
-
-    LayoutSemanticInfo semanticInfo;
-    semanticInfo.index = 0;
-    semanticInfo.space = 0;
+    const int kMaxResCount = 2;
```

Comment:
can you explain a little bit about meaning of those magic numbers?

Reply:
`kMaxResCount` is the size of an array.
This was used since we don't have a Slang `array` object and we are not supposed to be using stdlib.
Other parts of Slang code use c arrays, so I follow the pattern here.